x-pack/plugin/sql/src/test/java/org/elasticsearch/xpack/sql/plan/logical/command/sys/SysColumnsTests.java

```diff
@@ -130,6 +138,35 @@ public void testInNonDriverMode() {
         }
     }
 
+    public void testUnsignedLongFiltering() {
+        List<SqlVersion> versions = List.of(
+            SqlVersion.fromId(INTRODUCING_UNSIGNED_LONG.id - SqlVersion.MINOR_MULTIPLIER),
+            INTRODUCING_UNSIGNED_LONG,
+            SqlVersion.fromId(INTRODUCING_UNSIGNED_LONG.id + SqlVersion.MINOR_MULTIPLIER),
+            SqlVersion.fromId(Version.CURRENT.id)
+        );
+        for (Mode mode : List.of(Mode.JDBC, Mode.ODBC)) {
+            for (SqlVersion version : versions) {
+                List<List<?>> rows = new ArrayList<>();
+                SysColumns.fillInRows(
+                    "test",
+                    "index",
+                    loadMapping("mapping-multi-field-variation.json", true),
```

Comment:
Same here about loading the same mapping file multiple times. Isn't it enough to load the file once?

Reply:
Also `SysColumns#fillInRows()` behaves version-dependent.

Reply:
Fixed.